assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -112,16 +132,16 @@ function TextSets({ paneRef }) {
     overscan: 5,
   });
 
-  const sectionId = useMemo(() => `section-${uuidv4()}`, []);
-  const title = useMemo(() => __('Text Sets', 'web-stories'), []);
-
   const handleSelectedCategory = useCallback((selectedCategory) => {
     setSelectedCat(selectedCategory);
     localStore.setItemByKey(`${LOCAL_STORAGE_PREFIX.TEXT_SET_SETTINGS}`, {
       selectedCategory,
     });
   }, []);
 
+  const sectionId = `section-${uuidv4()}`;
```

Comment:
I added `useMemo` here because it doesn't seem like we want to regenerate a uuid every render